### ISSUE TITLE
Merge the 2 base cast damage boosting rings for each school of magic or get rid of them.

### ISSUE THREAD

**Issue by SpartanHexus** [TheGrandArchives/DS3CommunityPatch#31]
Right now if you want to do a pure caster for every school of magic, or even a dark build for every dark cast you would need 6 casting rings just to get the minimum acceptable damage a caster should have. If you merge the cast boosting rings into the stronger variants, that number will be reduced to 3. Example:

Bellowing dragoncest ring would also include young dragon ring effect, but you would not be able to equip the young dragon ring while having Bellowing ring on.

Or you could get rid of these rings requirements and boost magic by the same amount of damage the combined rings would do. Should be easy to apply a effect to a player that boosts all magic by 32% and then remove the ring effects.

**Comment by shieldpoke** [comment]
No lol. Given the changes already coming to the mages, this would definitely push them over the edge to being strong. And I would fucking hate for mages to be strong, given the best mage build is spamming the same 2-3 spells over and over again in the most boring way possible. Mages are PURE reactive, any initiative taken is essentially wasted. Melees are atleast able to pressure some what, mages just sit back and use soulmass. Unless you make significant changes to mages to make ANY other playstyle viable, any other buffs are completely and utterly wasted.

**Comment by shieldpoke** [comment]
Sure for faith boys, why not.

**Comment by SpartanHexus** [comment]
You can just side sprint against soul mass, or any spell really, unless they free aim, but then that's also super easy to roll. And needing less rings wouldn't do anything to solve or to make the issue with the "best mage builds" worse, it would only make it easier to build mages.

**Comment by shieldpoke** [comment]
yes, you can. BUT YOU HAVE TO HIT THEM EVENTUALLY. And then they can do it over and over again until you fuck it up and get hit by even one single soulmass. Yes it's super easy to roll, but it's also super easy to spam. It's just a patience game. Yes, you can side sprint everything they do, but you can also just roll every attack in the game by pure reaction times. Why do people get hit? It's because people aren't perfect, they WILL fuck up. And the entire mage archetype is built around that, and that's not fun. By making it easier to build mages, you are literally making the best mage build better.

**Comment by SpartanHexus** [comment]
Tell me how someone fucks up side spriting a soul mass. You can side sprint to force them to keep firing that soul mass before they want to, while you're at it, keep yourself close enough for a melee hit, you can also bring a crossbow. I'm sorry but nothing will change on that end regarding this suggestion even if they don't merge the rings. Because the rings are not responsible on how "The best mage build" work. 

"And the entire mage archetype is built around that, and that's not fun or interactive"

Being reactive is something that fits as the best playstyle on this thing's entire pvp. Or did you forgot that a battle between melees is usually "SPACING -> SOMEONE MISSES -> HIT -> SPACING -> SOMEONE MISSES -> HIT" This entire game requires patience, while i do agree its boring, mages not requiring 1-2 extra rings isn't going to make it worse.

**Comment by SpartanHexus** [comment]
Something that could help against soul mass spam on this patch is a big stamina cost increase upon casting the regular soul mass. Increasing fp cost would make it shit compared to other soul mass spells, but increasing the stamina cost enough to make reactive play harder with it would be interesting. The fp efficiency would still be there but they wouldn't be able to react to pressure as easily with it. Crystal homing soul mass would be better on that end but then it would cost too much FP for continuous use.

**Comment by shieldpoke** [comment]
>Tell me how someone fucks up side spriting a soul mass.

Are you kidding me? Look up the tournaments yourself. People DO and WILL fuckup side sprints. If you play unlocked and turn ur back and shoot one soul mass at a time, you can fire like 100 soul masses INDIVIDUALLY over the course of a match. I've literally hosted a tournament where a mage did exactly that against an equally good opponent. IT was phina vs meryx, which if you are familiar with the top players, should ring a bell and mean something

"SPACING -> SOMEONE MISSES -> HIT -> SPACING -> SOMEONE MISSES -> HIT" 

The difference here is that YOU CAN ACTUALLY PRESSURE. Just look up Gabri pressuring with a greatsword or a caestus. He runs them down or bdubsz pressuring with a curved sword. You absolutely CAN be aggressive.You literally cannot pressure with mage spells, the only thing you can do is wait for them to attack.

Stamina cost? Are you joking? Unless you make it cost like 50% of your stam bar, this will do absolutely nothing. It's not like mages use shields or make use of anything stam intensive anyway.

**Comment by SpartanHexus** [comment]
No, i'm not joking and i was thinking of more than 50% of a stamina bar, great heal level of stamina consumption, make it hard to even roll after the cast. The problem is how easy is to spam and rely on that spell alone and how hard it is to punish it. To change that we can increase fp consumption for a long term result on the fight or massively increase sp consumption for a immediate consequence. Alternatively changing soul mass animation to something slower would also work, like the chamaleon/pestilent mist/cast light animation.

I haven't been given a reason on how not needing rings would make it worse, it will make building for any kind of magic easier, not just "The best magic playstyle". It isn't as if letting rings are they are or changing them is gonna encourage or discourage the passive playstyle. The only thing that will ever do it is to change the very spells that encourage people to do so.

**Comment by shieldpoke** [comment]
Then ur killing the spell altogether. gg
